src/Dialect/ONNX/ONNXOpsHelper.hpp

```diff
@@ -17,14 +17,32 @@
 #include "mlir/Dialect/Traits.h"
 #include "mlir/IR/AffineExpr.h"
 #include "mlir/IR/AffineMap.h"
+#include "mlir/IR/Block.h"
```

Comment:
I would have preferred to have the fewest possible includes in commonly used header files like this one, so .cpp files don't pull in excess indirect includes. Is it an option to make only the .cpp files that need them include Block.h, SetVector.h, etc?

Reply:
Agreed, this can be done in a later PR as it will be small localized changes.